### ISSUE TITLE
make sharing to twitter easy

### ISSUE THREAD

**Issue by techieshark** [codeforamerica/MuniciPal#94]
There should be a share button on each item. Disqus provides this but it is not immediately obvious. The button should be visible before Disqus is expanded. When you tweet, it should link to the event item the button was attached to, it should have a #civicmesa hashtag, and it should @-mention the councilmember (who serves the selected district) on twitter.

**Comment by michaelhwan** [comment]
Why not just add the tweet button using Javascript?

**Comment by buckleytom** [comment]
@michaelhwan i think we tried this at some point. it was a fun challenge for a number of reasons. although i don't think we ever had the time to make it work.

**Comment by buckleytom** [comment]
@michaelhwan give it a shot! it might be easier to try this outside of the app/environment here first and then think about how you would slot it in.

**Comment by michaelhwan** [comment]
@techieshark @tombuckley I don't think this is needed. Adding more social media features is nice but it can clutter things up on the webpage.

**Comment by techieshark** [comment]
If done well it need not be cluttered.

http://www.awwwards.com/sites/the-lobster
